dev/ansible/04_upload_files.yml

```diff
@@ -56,7 +56,7 @@
   tags: example-upload
 
 - name: Verify that file exist on target `/one/some/prefix/iris-file.csv`
-  shell: mc cat minio-container/one/some/prefix/iris-file.csv
+  shell: mc ls minio-container/one/some/prefix/iris-file.csv
```

Comment:
Are you sure that you can run `ls` on object(`iris-file.csv`), I thought `ls` is only for buckets and directories under bucket? [doc](https://docs.min.io/docs/minio-client-complete-guide.html#ls)

Reply:
```
ls          list buckets and objects
cat         display object contents
```
I guess object means the same for both **ls** and **cat** commands. I checked the o/p by adding a debug message and it worked, i can add debug messages so as we have clarity based on the o/p displayed.